meta-refkit/conf/bblayers.conf.sample

```diff
@@ -35,6 +35,10 @@ REFKIT_LAYERS += "##OEROOT##/../meta-openembedded/meta-python"
 # REFKIT_LAYERS += "##OEROOT##/../meta-openembedded/meta-xfce"
 # REFKIT_LAYERS += "##OEROOT##/../meta-openembedded/toolchain-layer"
 
+# Demo layer, disabled by default. To enable, uncomment the line below
+# and increment LCONF_VERSION by one.
+# REFKIT_LAYERS += "##OEROOT##/../meta-refkit-extra"
```

Comment:
I suspect that this comment might become misleading once bblayers.conf.sample turns into conf/bblayers.conf. A developer editing that bblayers.conf the might uncomment the line and bump LCONF_VERSION, which wouldn't be right.

Do we need this change in bblayers.conf.sample at all?

A developer who wants the extra layer can do "bitbake-layers add-layer <path to meta-refkit-extra>".